sdl_viewer/src/main.rs

```diff
@@ -194,6 +197,62 @@ impl NodeView {
     }
 }
 
+// Keeps track of the nodes that were requested in-order and loads then one by one on request.
+struct NodeViewContainer {
+    node_views: HashMap<octree::NodeId, NodeView>,
+    unloaded_nodes: VecDeque<octree::NodeId>,
+    queued: HashSet<octree::NodeId>,
+}
+
+impl NodeViewContainer {
+    fn new() -> Self {
+        NodeViewContainer {
+            node_views: HashMap::new(),
+            unloaded_nodes: VecDeque::new(),
```

Comment:
`unloaded_nodes` -> `queue`? Unloaded sounds like they have been removed, but this is the same as `queued` just in the correct ordering, i.e. the queue, right?